pkg/virt-controller/watch/vmi/lifecycle.go

```diff
@@ -1086,3 +1099,16 @@ func preparePodPatch(oldPod, newPod *k8sv1.Pod) *patch.PatchSet {
 		patch.WithReplace("/status/conditions", newPod.Status.Conditions),
 	)
 }
+
+func handleEvictionInProgressAnnotation(vmi *virtv1.VirtualMachineInstance) *virtv1.VirtualMachineInstance {
+	const aValue = "kubevirt"
```

Comment:
Make it say `"true"` instead?

Reply:
in past in `descheduler.go` (https://github.com/kubevirt/kubevirt/pull/15423/files#diff-64139376a32bee7bb7b3ba83308fcf73818ef41d4455d9f043db04d5ee8fefbeL61) we were using `kubevirt`.
AFAIK it's just a placehold and the descheduler code is only checking the presence of the key completely ignoring its value. So yes, I think we can safely change if we think it will be more consistent.